python/extractor/tsg-python/BUILD.bazel

```diff
@@ -12,5 +12,5 @@ codeql_rust_binary(
     visibility = ["//visibility:public"],
     deps = all_crate_deps(
         normal = True,
-    ) + ["//python/extractor/tsg-python/tree-sitter-python"],
```

Comment:
nit: `tspy` would probably have been better to avoid confusion with the Tool Status Page (TSP), but not a blocker.